P5/Source/Specs/att.locatable.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<classSpec xmlns="http://www.tei-c.org/ns/1.0" module="tei" type="atts" ident="att.locatable">
+  <desc versionDate="2020-01-15" xml:lang="en">provides attributes that specify locational information.</desc>
+  <attList xmlns:rng="http://relaxng.org/ns/structure/1.0">
+    <attDef ident="where" usage="opt">
+      <desc versionDate="2020-01-15" xml:lang="en">The <att>where</att> specifies a location by using a URI to another resource.</desc>
+      <datatype maxOccurs="unbounded"><dataRef key="teidata.pointer"/></datatype>
+    </attDef>
+  </attList>
+      <exemplum xml:lang="en">
+        <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#MONLONDON">
+          <!--In the teiHeader-->
+          <listPlace>
+            <place xml:id="PAUL1">
+              <placeName>Paul's Chain</placeName>
+            </place>
+            <place xml:id="STPA3">
+              <placeName>Paul's Churchyard</placeName>
+            </place>
+          </listPlace>
+          <!--Elsewhere, in the body of the document-->
+          <p>This Shew hauing tendred this ſeruice to my
+            Lord vppon the Water, is after to be conueyed a
+            Shore, and in conuenient place employd for adorning the reſt of the Triumph.
+            After my Lord Maiors landing, and comming paſt <ref target="#PAUL1">Paules Chaine</ref>
+            firſt attends for his Honor in <ref target="#STPA3">Pauls Church–yarde</ref>
+            a beautifull Spectacle, called the
+            Temple of Honor
+            <!--More prose describing the scene-->
+          </p>
+          <move where="#PAUL1 #STPA3"/>
+          <div>
+            <head>The ſpeech of Troynouant.</head>
+            <sp>
+              <lg>
+                <l>HIſtory, Truth, and Vertue ſeeke by name,</l>
+                <!--...-->
+              </lg>
+            </sp>
+          </div>
+        </egXML>
+      </exemplum>
+ 
+  <remarks versionDate="2020-01-15" xml:lang="en">
+    <p>This attribute class provides <att>where</att> for use in pointing to a URI containing a canonical name or information about the locational context of an <gi>event</gi> or a <gi>move</gi> on stage. It can be used for pointing out blocking cues, as in stage left, right, or center, where these locations are defined in the <gi>teiHeader</gi> or an external source. It may also be used to point to a canonical <gi>place</gi> defined in a <gi>listPlace</gi>.</p>
```

Comment:
maybe: "… for use in pointing to a resource containing a canonical …"

Reply:
Yes--that's better! Edited.

Reply:
I'm also adding a bit more to the remarks to make the drama context a little clearer for `<move>`.